build-support/scripts/up

```diff
@@ -12,12 +12,26 @@ docker run -d --name ad1 --rm \
     -p 48005:48005 \
     -e "NUODB_DOMAIN_ENTRYPOINT=ad1" \
     ${IMG_NAME} nuoadmin
+
+COMMAND="docker exec -i ad1 nuocmd show domain"
+echo -n "Wating for AP to start..."
+while ! $COMMAND 2>/dev/null | grep -q "Connected"; do
+    echo -n "."
+    sleep 2
+done
+echo -e "\nAP started!"
+echo
+
+# Apply limited license
+echo -n "Applying license.. "
+docker exec -it -e "NUODB_LIMITED_LICENSE_CONTENT=$NUODB_LIMITED_LICENSE_CONTENT" ad1 /bin/bash -c 'echo "$NUODB_LIMITED_LICENSE_CONTENT" > ~/nuodb.lic && nuocmd set license --license-file ~/nuodb.lic && rm ~/nuodb.lic'
```

Comment:
You need to base64 decode the the license content:

```
docker exec -it ... /bin/bash -c 'echo $NUODB_LIMITED_LICENSE_CONTENT | base64 -d > ~/nuodb.lic && ...'
```

Reply:
That worked. Thanks.